packages/openapi3/package.json

```diff
@@ -64,7 +64,8 @@
     "@typespec/compiler": "workspace:~",
     "@typespec/http": "workspace:~",
     "@typespec/openapi": "workspace:~",
-    "@typespec/versioning": "workspace:~"
+    "@typespec/versioning": "workspace:~",
+    "@typespec/xml": "workspace:~"
```

Comment:
wondering if we can make this an optional dependency(or just dev as optional always get installed anwway) and we just try to load the library if present.

Reply:
reopening this as we have had similar problem recently with http library needing a dep on stream.
I think you also should add 
```
"peerDependenciesMeta": {
    "@typespec/xml": {
      "optional": true
    }
  },
```
and make sure in the code you dynamically handle loading the library(if its not present just ignore)  otherwise we are now requiring everybody using openapi to also install the xml library which they most likely don't need